### DISCUSSION TITLE
Windows build?

### DISCUSSION THREAD

**Discussion by ghost** [smcameron/space-nerds-in-space#313] [Q&A] [answered]
Hello, I really love this game but is there a way to build this on Windows 11? That way I can play with my linux brother at the same time :)

**Comment by smcameron** [comment, marked as answer]
I have seen it run on Window System for Linux (WSL), on Windows 10, however the graphics was emulated in software with no access to the GPU so performance was not very good.  It might work with WSLg which purports to allow using the GPU ... see: https://www.youtube.com/watch?v=b1YBx1L8op4  ... but I have no actual experience with WSLg, so I don't know, I'm just repeating words that I read on the internet.  Good luck, and let me know if you have any success with it.

**Reply by ghost** [reply]
Sadly it doesn't seem to be working.. It just crashes my computer while setting up. I do see a whole bunch of things now to run Linux games on Windows, but I don't think I have much storage space for some of the good ones.

I'm very new to programming in C++ (my first programming language) as a hobby and I think it may be possible to compile the game for Windows? Unless you use linux-specific header files I think. But I don't have any experience with makefiles so I'm at a loss for that...

**Reply by smcameron** [reply]
Porting it to run natively on windows would be a big job.  I don't think it's just a simple recompile.  It uses pthreads extensively and there are undoubtedly other things that would need to be fixed.  It was not written to be cross platform.

**Comment by smcameron** [comment]
Speaking of remote opengl, I came across this: https://virtualgl.org/About/Background  I guess it is akin to what OBS does... render on the machine with the GPU, screen capture, compress and transmit to the remote machine.
